DOCKER_SETUP.md

```diff
@@ -0,0 +1,244 @@
+# F3 Nation Slack Bot - Docker Development Setup
+
+This setup includes automatic database schema creation using the F3-Data-Models repository.
+
+## Quick Start
+
+0. For now.. Ensure apparmor or selinux is disabled on your devel box
+```
+# ubuntu / debian
+sudo apparmor_status
+# fedora / rhel
+getenforce
+```
+
+
+1. **Create your `.env` file** with your Slack credentials:
+   ```bash
+   cp .env.example .env  # if available
+   # Edit .env with your values
+   ```
+   
+   ```bash
+   # Database Configuration  
+   export DATABASE_USER=postgres
+   export DATABASE_PASSWORD=postgres
+   export DATABASE_SCHEMA=f3_schema
+   export POSTGRES_PASSWORD=$DATABASE_PASSWORD
+   export DATABASE_SCHEMA=f3_schema
+   export POSTGRES_DB=f3_db
+
+   # Slack Configuration
+   export SLACK_BOT_TOKEN=xoxb-your-bot-token
+   export SLACK_SIGNING_SECRET=your-signing-secret
+   ```
+
+2. **Choose your development approach:**
+
+   ### Option A: Run App Locally (Recommended for Development)
+   Running the app locally is much easier if using debugging like pdb.
+   ```bash
+   # Start only database and initialization (no app container)
+   docker-compose up -d
+   
+   # In a separate terminal, run the app locally
+   # Assumes your virtualenv is created and sourced.
+   
+   # Set local environment variables (these override compose.yml for local app)
+   export DATABASE_HOST=localhost      # Connect to Docker DB from outside
+   export LOCAL_DEVELOPMENT=true       # Enable development mode + skip data restoration
+   source .env                         # Load your Slack credentials
+   
+   # Run the app locally
+   nodemon --exec "python main.py" -e py
```

Comment:
This will need to be "poetry run python main.py" if they've followed the steps of installing dependencies to a virtual environment